### DISCUSSION TITLE
Issues with fastapi response models

### DISCUSSION THREAD

**Discussion by Morph-Tollon** [fastapi-users/fastapi-users#1383] [General]
I am famously bad at following docs so i have opened up a discussion here to try and fix my problem, I followed the docs for using sqlalchemy and postgres, and I'm getting the familiar` Invalid args for response field!` for :

```
app.include_router(
    fastapi_users.get_auth_router(auth_backend), prefix="/auth/cookie", tags=["auth"]
)

app.include_router(
    fastapi_users.get_register_router(UserRead, UserCreate),
    prefix="/auth",
    tags=["auth"],
)
app.include_router(
    fastapi_users.get_reset_password_router(),
    prefix="/auth",
    tags=["auth"],
)
app.include_router(
    fastapi_users.get_verify_router(UserRead),
    prefix="/auth",
    tags=["auth"],
)
app.include_router(
    fastapi_users.get_users_router(UserRead, UserUpdate),
    prefix="/users",
    tags=["users"],
)
```
Is there a way to fix this without going under the hood, is it an issue I have created or something in the module itself?

Edit: fixed formatting

**Comment by frankie567** [comment]
More information will be needed in order to help you. At what point do you get this error? Sharing the full stack trace could also help.
